packages/docs/components/SideNav/SideNav.tsx

```diff
@@ -30,7 +30,7 @@ export const SideNav: React.FC = () => {
         <SideNavGroup title="Introduction">
           <SideNavLink href="/">Getting Started</SideNavLink>
 
-          <Link title="CodeSandbox Example" url={CodeSandboxUrl} />
+          <Link title="CodeSandbox Example" url={String(CodeSandboxUrl)} />
```

Comment:
wondering if we should add `""` in case `CodeSandboxUrl` is undefined or null? I think if `CodeSandboxUrl` is `undefined` or `null`, we could get `'undefined'` or `'null'`